fastjet/internal/predicates/orientation_test.go

```diff
@@ -30,3 +32,101 @@ func TestOrientation(t *testing.T) {
 		}
 	}
 }
+
+func TestSimpleVsRobustOrientation(t *testing.T) {
+	tests := []struct {
+		x1, y1, x2, y2, x, y float64
+		simple               OrientationKind
+		robust               OrientationKind
+	}{
+		{2.1, 2.1, 1.1, 1.1, 0.1, 0.1, IndeterminateO, Colinear},
+		{2.1, 2.1, 1.1, 1.1, 100.1, 100.1, IndeterminateO, Colinear},
+		{2.1, 2.1, 1.1, 1.1, 1000.1, 1000.1, IndeterminateO, Colinear},
+	}
+	for _, test := range tests {
+		o := simpleOrientation(test.x1, test.y1, test.x2, test.y2, test.x, test.y)
+		if o != test.simple {
+			t.Errorf("x1 = %v, y1 = %v, x2 = %v, y2 = %v, x = %v, y = %v, want.Simple = %v. got= %v\n", test.x1, test.y1, test.x2, test.y2, test.x, test.y, test.simple, o)
+		}
+		o = robustOrientation(setBig(test.x1), setBig(test.y1), setBig(test.x2), setBig(test.y2), setBig(test.x), setBig(test.y))
+		if o != test.robust {
+			t.Errorf("x1 = %v, y1 = %v, x2 = %v, y2 = %v, x = %v, y = %v, want.Robust = %v. got= %v\n", test.x1, test.y1, test.x2, test.y2, test.x, test.y, test.robust, o)
+		}
+		o = matOrientation(test.x1, test.y1, test.x2, test.y2, test.x, test.y)
+		if o != test.simple {
```

Comment:
shouldn't you test against `test.robust` instead?
I'd expect the `gonum/mat` routines to be robust :)
(c.f. my comment above)